src/components/grid-aware/OneParagraphBlock/OneParagraphBlock.module.css

```diff
@@ -0,0 +1,125 @@
+/* Outer grid */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-black);
+  grid-column: 2 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 140px 0 93px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 80px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaLeft {
+  grid-column: 2 / span 4;
+}
+
+.gridAreaRight {
+  grid-column: 7 / span 5;
+}
+
+
+/* Title block */
+
+.title {
+  font: var(--font-headline);
+}
+
+
+/* Jobs block */
+
+.jobsWrapper {
+  margin-top: 0;
+}
+
+@media (--tablet-and-down) {
+  .jobsWrapper {
+    margin-top: 40px;
+  }
+}
+
+.jobs *:first-child {
+  margin: 0;
+}
+
+.jobs *:not(:first-child) {
+  margin-top: 20px;
+}
+
+.jobs *:nth-child(odd) {
+  font: var(--font-subtitle);
+}
+
+.jobs *:nth-child(even) {
+  font: var(--font-body-medium);
+}
```

Comment:
Is the asterisk necessary to select the elements with the .jobs class?

Reply:
No, they are not. The pseudo-classes are enough. You may also want to use the direct child select `>` so that you don't select even more nested elements, since these will select any element at any depth of the hierarchy.